L1Trigger/L1THGCal/src/backend/HGCalSortingTruncationImpl_SA.cc

```diff
@@ -0,0 +1,23 @@
+#include "L1Trigger/L1THGCal/interface/backend/HGCalSortingTruncationImpl_SA.h"
+
+HGCalSortingTruncationImplSA::HGCalSortingTruncationImplSA() {}
+
+void HGCalSortingTruncationImplSA::sortAndTruncate_SA(
+    const std::vector<l1thgcfirmware::HGCalMulticluster>& inputMulticlusters,
+    std::vector<l1thgcfirmware::HGCalMulticluster>& outputMulticlusters,
+    const l1thgcfirmware::SortingTruncationAlgoConfig& configuration) const {
+  for (const auto& multicluster : inputMulticlusters) {
```

Comment:
Could call `reserve` on `outputMulticlusters` before pushing since the size is known

Reply:
thanks, done.